docs/options.md

```diff
@@ -74,15 +74,31 @@ comment: |
 
 | Name     | Default Value | Workflow file | Config File        |
 | -------- | ------------- | ------------- | ------------------ |
-| `commit` | n/a           | :x:           | :heavy_check_mark: |
+| `commit` | `true`        | :x:           | :heavy_check_mark: |
 
-The _commit_ option can be used to configure the commits created by the Action.
-Because it is a complex option, it can only be configured in a configuration
-file.
+The _commit_ option can be used to either 1) disable commits or 2) configure the
+commits created by the Action. Because it is a complex option, it can only be
+configured in a configuration file.
```

Comment:
This is still not entirely up-to-date, the option can now also be used in the workflow file, but only as a boolean